settings/provisioner/virsh/virsh.spec

```diff
@@ -38,6 +49,7 @@ subparsers:
                   cleanup:
                       action: store_true
                       help: Clean given system instead of provisioning a new one
+                      requires_only: [host, ssh-user,  ssh-key, topology]
```

Comment:
+1
Haven't seen the implementation yet, but liked the idea!
Please document it in the project's README file.